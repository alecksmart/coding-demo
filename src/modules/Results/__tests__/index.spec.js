/* eslint-disable import/no-unresolved */
/* eslint-disable import/order */
import React from 'react';
import store from '../../../store';
import shallowWithStore from 'shallowWithStore';
import Results from '../index';

const context = {
  pointsGame: {
    started: false, isGameOver: false, bonuses: 0, total: 30, highest: 0, log: [{ type: 'B' }],
  },
};

describe('>>> Client entry point', () => {
  it('+++ should capture the snapshot of Cell', () => {
    const renderedValue = shallowWithStore(
      <Results store={store} />,
      context,
    );
    expect(renderedValue).toMatchSnapshot();
  });
});
