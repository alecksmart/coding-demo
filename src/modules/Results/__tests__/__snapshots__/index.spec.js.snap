// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`>>> Client entry point +++ should capture the snapshot of Cell 1`] = `ShallowWrapper {}`;
